### DISCUSSION TITLE
Images in multiple directories

### DISCUSSION THREAD

**Discussion by dig-lib** [cantaloupe-project/cantaloupe#522] [Q&A]
I'm a complete rookie and not a techie. I've just completed the IIIF course and have a big project to start. It requires IIIF image server so I'm testing digilib after cantaloupe. Cantaloupe is running smoothly on my local machine. The only thing I couldn't figure out so far was how to make it see more than one directory on my file system like 
/images/book1/pages.tif 
/images/book2/pages.jpg
I tried change everything I could think of in cantaloupe.properties file as FilesystemSource.BasicLookupStrategy.path_prefix: different separators, suffixes, etc. but nothing seems to work. I'd appreciate any advice.

**Comment by adolski** [comment]
Hi,

Set `FilesystemSource.BasicLookupStrategy.path_prefix = /images/`

URL identifiers will then be in the form: `book1%2Fpages.tif` and `book2%2Fpages.jpg` (`%2F` being a URL-encoded slash)

**Reply by dig-lib** [reply]
Thanks, I finally got the idea.

**Reply by micahchoo** [reply]
@adolski is it possible to do this for nested subfolders for any number of levels?
